ExampleSonarQubeIssues.cs

```diff
@@ -9,11 +9,16 @@ namespace sonarqube_nemo_on_appveyor
     // The full list of C# SonarQube rules is at http://dist.sonarsource.com/reports/coverage/rules/csharpsquid_rules_coverage.html
     public class ExampleSonarQubeIssues
     {
-        private void EmptyMethod()
+        void EmptyMethod()
         {
 
         }
 
+        void InfiniteRecusrion()
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a way to break out of this method's recursion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS2190)

Reply:
ha ha! I totally hadn't spotted that! I've committed a fix.

There is the [Spell Checker plugin](https://github.com/webdizz/sonarqube-spellcheck), but it only works with Java, and I don't think it works with the latest version of SonarQube,